containers/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Base image can be found here: https://hub.docker.com/r/jrottenberg/ffmpeg/
+# There are options for selecting a different distro.
+FROM jrottenberg/ffmpeg:3.3

```

Comment:
use the full name: `docker.io/jrottenberg/ffmpeg:3.3-ubuntu`
also, why not `4.1-vaapi`?

Reply:
Using `docker.io/tnk4on/yt-dlp` now, which might not have this, but it exposes ffmpeg and yt-dlp.